clients/drcachesim/simulator/caching_device.h

```diff
@@ -245,8 +249,8 @@ class caching_device_t {
     init_blocks() = 0;
 
     int associativity_;
-    int block_size_; // Also known as line length.
-    int num_blocks_; // Total number of lines in cache = size / block_size.
+    long int block_size_; // Also known as line length.
```

Comment:
This may need a mention in api/docs/release.dox (line 127) as it could break or require changes in third-party code deriving from this: we don't consider non-interface changes as breaking enough to go bump the major version so I would add a new list in between the two existing lists of "minor compatibility changes" (as was done for prior releases below in that file: e.g. line 574).